api-test/src/main/resources/pms/OIDCClientEnhancement/DeactivateOIDCClient/DeactivateOIDCClientNegTC.yml

```
@@ -0,0 +1,265 @@
+DeactivateOIDCClientV2NegTC:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)